src/main/scala/com/microsoft/hyperspace/actions/RefreshDeleteAction.scala

```diff
@@ -44,30 +42,13 @@ class RefreshDeleteAction(
     spark: SparkSession,
     logManager: IndexLogManager,
     dataManager: IndexDataManager)
-    extends RefreshActionBase(spark, logManager, dataManager)
+    extends RefreshDeleteActionBase(spark, logManager, dataManager)
     with Logging {
 
   final override protected def event(appInfo: AppInfo, message: String): HyperspaceEvent = {
     RefreshDeleteActionEvent(appInfo, logEntry.asInstanceOf[IndexLogEntry], message)
   }
 
```

Comment:
`validate` is moved to (new) class `RefreshDeleteActionBase`.

Reply:
Cool, thanks for letting us know.